.github/workflows/tests.yml

```
@@ -18,6 +18,11 @@ jobs:
       - name: Validate Scala
         uses: ./.github/actions/scala-test/
 
+      - name: Upload Scala coverage to Codecov
+          uses: codecov/codecov-action@v4
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)